tests/tasks/test_generate.py

```diff
@@ -32,3 +34,23 @@ def test_generate_custom_generator(self):
 
         self.assertIsInstance(generator_class, mock_generator.MockGenerator)
 
+    def test_generate_update_datapackage_sources(self):
+        """Test that GeneratorManager task updates datapackage sources"""
+
+        def empty_datapackage_sources(datapkg_path, datapkg):
+            with io.open(datapkg_path, mode='w+', encoding='utf-8') as datapkg_file:
+                datapkg.metadata['sources'] = []
```

Comment:
I used `datapackage-py 0.6.1.` which is the latest version according to pip and [datapackage-py repo](https://github.com/frictionlessdata/datapackage-py/blob/aa483ce04b41b1218d2f387040c06e2ed69e0644/setup.py#L37).